### PR TITLE
Blog Post: Long-Term Support for Stuart Bowman

### PR DESCRIPTION
**Pull request by dergigi**
Ready for final review and sanity check.

**Build preview:**

- [/blog/stuart-bowman-receives-lts-grant](https://os-website-git-stuart-lts-open-sats.vercel.app/blog/stuart-bowman-receives-lts-grant)

**Social preview:**

Screenshot 2024-04-12 at 22 58 54

### CODE REVIEW

**Review by lovvtide — CHANGES_REQUESTED** [review]
I would just update the bullet points with the new links like this:

---

- [Satellite Public Node](https://github.com/satellite-earth/public-node): A "nostr node" with enhanced relay capabilities, providing a dedicated backend for communities.
- [Satellite Private Node](https://github.com/satellite-earth/private-node): A local nostr relay and media proxy to boost speed, privacy, and network redundancy.
- [Satellite Control](https://github.com/satellite-earth/dashboard-ui): A control panel that allows non-technical users to operate their relay and media archive.
- [Satellite Desktop](https://github.com/satellite-earth/desktop): A cross-platform desktop app to easily run satellite nodes on consumer hardware.
- [Satellite CDN](https://github.com/lovvtide/satellite-cdn): A Blossom-compatible CDN server that handles media uploads, downloads, user authentication, lightning payments, and file indexing.

File: data/blog/stuart-bowman-receives-lts-grant.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+The overarching goal of Satellite is to strengthen the nostr network by
+strengthening the various communities that make up the network. The “Satellite
+Stack” will allow communities to be self-reliant and thus robust, improving the
+relay system for greater diversity and functionality in the process. In the end,
+this will expand the network's capabilities and increase its value and activity.
+
+Stuart will continue to work on the Satellite Stack and related components, such
+as:
+
+- [Satellite Node](https://github.com/lovvtide/satellite-node): A nostr node with enhanced relay capabilities, providing a dedicated backend for communities.
```
Comment (Arvin21M) [inline data/blog/stuart-bowman-receives-lts-grant.mdx:27]:
```suggestion
- [Satellite Public Node](https://github.com/satellite-earth/public-node): A "nostr node" with enhanced relay capabilities, providing a dedicated backend for communities.
```

File: data/blog/stuart-bowman-receives-lts-grant.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+- [Satellite Control](https://github.com/lovvtide/satellite-control): A control panel that allows non-technical users to operate their relay and media archive.
```
Comment (Arvin21M) [inline data/blog/stuart-bowman-receives-lts-grant.mdx:28]:
```suggestion
- [Satellite Control](https://github.com/satellite-earth/dashboard-ui): A control panel that allows non-technical users to operate their relay and media archive.
```

File: data/blog/stuart-bowman-receives-lts-grant.mdx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+- [Satellite Desktop](https://github.com/lovvtide/satellite-desktop): A cross-platform desktop app to easily run satellite nodes on consumer hardware.
```
Comment (Arvin21M) [inline data/blog/stuart-bowman-receives-lts-grant.mdx:29]:
```suggestion
- [Satellite Desktop](https://github.com/satellite-earth/desktop): A cross-platform desktop app to easily run satellite nodes on consumer hardware.
```

File: data/blog/stuart-bowman-receives-lts-grant.mdx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+- [Satellite CDN](https://github.com/lovvtide/satellite-cdn): A Blossom-compatible CDN server that handles media uploads, downloads, user authentication, and file indexing.
```
Comment (Arvin21M) [inline data/blog/stuart-bowman-receives-lts-grant.mdx:30]:
```suggestion
- [Satellite CDN](https://github.com/lovvtide/satellite-cdn): A Blossom-compatible CDN server that handles media uploads, downloads, user authentication, lightning payments, and file indexing.
```

File: data/blog/stuart-bowman-receives-lts-grant.mdx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
```
Comment (Arvin21M) [inline data/blog/stuart-bowman-receives-lts-grant.mdx:31]:
```suggestion
- [Satellite Private Node](https://github.com/satellite-earth/private-node): A local nostr relay and media proxy to boost speed, privacy, and network redundancy.

```

**Review by Arvin21M — CHANGES_REQUESTED** [review]

**Comment by dergigi** [comment]
Apologies @Arvin21M, I made the same changes that you made without checking whether you already updated stuff. Sorry 😕

**Comment by Arvin21M** [comment]
> Apologies @Arvin21M, I made the same changes that you made without checking whether you already updated stuff. Sorry 😕
> 
> Edit: Reverted my changes & applied yours.

No problem @dergigi, just happy this is done 😃

**Comment by dergigi** [comment]
Cover image added, ready for final review.

**Review by Arvin21M — APPROVED** [review]
Awesome header image! All the links work and point to the right place. Text matches final draft. Good to go 🚀

**Comment by btcschellingpt** [comment]
Add a definition of CDN in the opening paragraph to provide context for those that don't already know that.

**Pull request merged**